capstone-api/models/User.js

```diff
@@ -1,128 +1,114 @@
+    userPreferences.set("topArtists", []);
+    userPreferences.set("username", username);
+
+    try {
+      await user.signUp();
+    } catch (error) {
+      return `Error with ${username} registering account`;
     }
 
-    static async logUserIn(usernameValue, passwordValue) {
-        Parse.User.enableUnsafeCurrentUser();
-        const loggedInUser = await Parse.User.logIn(usernameValue, passwordValue);
-        return loggedInUser;
+    try {
+      await userPreferences.save();
+    } catch (error) {
+      return `Error with ${username} saving preferences`;
     }
 
-    static async registerUser(userValue) {
-      let {username, password, email, spotifyURL, imageURL } = userValue;
-      let user = new Parse.User();
-
-      user.set("username", username);
-      user.set("password", password);
-      user.set("email", email);
-      user.set('followers', []);
-      user.set("appPassword", password);
-      user.set('spotifyURL', spotifyURL);
-      user.set('imageURL', imageURL);
-
-      let userPreferences = new Parse.Object('Preferences');
-      userPreferences.set('topGenres', []);
-      userPreferences.set('topArtists', []);
-      userPreferences.set('username', username);
-
-      try {
-          await user.signUp();
-          await userPreferences.save();
-          return "User created!";
-      } catch (error) {
-          return error.message;
-      }
+    return true;
   }
 
-    static async post(username, trackId) {
-        let Post = new Parse.Object('Post');
-        Post.set('username', username);
-        Post.set('trackId', trackId);
-        try {
-          await Post.save();
-          return true;
-        } catch (error) {
-          return false;
-        };
-    };
-
-    static async posts() {
-        const query = new Parse.Query('Post');
-        try {
-          let todos = await query.find();
-            return todos;
-        } catch (error) {
-          return {};
-        };
-    };
-    
-    static async timeline(username) {
-      const query = new Parse.Query('Post');
-      query.equalTo("username", username);
-      query.descending("createdAt");
-      let result = await query.find();
-      return result;
-  };
-
-    static async getPost(username, trackId) {
-      const query = new Parse.Query('Post');
-      query.equalTo("username", username);
-      query.equalTo("trackId", trackId);
-      try {
-        let posts = await query.find();
-        return posts[0];
-      } catch (error) {
-        return {};
-      };
-    };
-
-    static async getUsers() {
-      const query = new Parse.Query('User');
-      try {
-        let users = await query.find();
-        return users;
-      } catch (error) {
-        return {};
-      };
-    }
-
-    static async followUser(currUsername, followUsername) {
-      let user = Parse.User.current();
-      user.addUnique('followers', followUsername);
-      await user.save();
+  static async post(username, trackId) {
```

Comment:
can you give this a better function name that is more descriptive about what it does?